src/emulator/modules/SceAppUtil/SceAppUtil.cpp

```diff
@@ -149,40 +150,37 @@ EXPORT(int, sceAppUtilResetCookieWebBrowser) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceAppUtilSaveDataDataRemove, emu::SceAppUtilSaveDataFileSlot *slot, emu::SceAppUtilSaveDataRemoveItem *files, unsigned int fileNum, SceAppUtilSaveDataMountPoint *mountPoint) {
+static std::string construct_savedata0_path(const std::string &data, const char *ext = "") {
+    return device::construct_normalized_path(VitaIoDevice::savedata0, data, ext);
+}
+
+static std::string construct_slotparam_path(const unsigned int data) {
+    return construct_savedata0_path(std::string("SlotParam_" + std::to_string(data)), "bin");
+}
+
+EXPORT(int, sceAppUtilSaveDataDataRemove, SceAppUtilSaveDataFileSlot *slot, emu::SceAppUtilSaveDataRemoveItem *files, unsigned int fileNum, SceAppUtilSaveDataMountPoint *mountPoint) {
```

Comment:
Ahh! `emu::` missing here too! You can use psp2 header if the struct does not contain a pointer or a struct that contains a pointer (because the sizes will be wrong). SceAppUtilSaveDataFileSlot has a pointer :/